packages/ui/src/working-groups/components/WorkingGroupImagePlaceholder.tsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Colors, Transitions } from '../../common/constants'
+
+export interface WGImagePlaceholderProps {
```

Comment:
```suggestion
export interface WorkingGroupImageProps {
```

Try to avoid abbreviations ;)